packages/dds/register-collection/src/consensusRegisterCollection.ts

```diff
@@ -67,7 +67,7 @@ interface IRegisterOperation {
 }
```

Comment:
So the "current" type (as of before this PR) has `refSeq: number | undefined`. The "old" type (now used again in this PR) has `refSeq: number`.  Do we need to account for the `undefined` case or not?

Also, that comment about the semantics of `refSeq` should be brought over too.